src/Controls/Geosearch.js

```diff
@@ -386,19 +229,30 @@ export var Geosearch = L.Control.extend({
     L.DomEvent.disableClickPropagation(this._wrapper);
 
     // when mouse moves over suggestions disable scroll wheel zoom if its enabled
-    L.DomEvent.addListener(this._suggestions, 'mouseover', function (e) {
+    L.DomEvent.addListener(this._geosearchClass._suggestions, 'mouseover', function (e) {
       if (map.scrollWheelZoom.enabled() && map.options.scrollWheelZoom) {
         map.scrollWheelZoom.disable();
       }
     });
 
     // when mouse moves leaves suggestions enable scroll wheel zoom if its disabled
-    L.DomEvent.addListener(this._suggestions, 'mouseout', function (e) {
+    L.DomEvent.addListener(this._geosearchClass._suggestions, 'mouseout', function (e) {
       if (!map.scrollWheelZoom.enabled() && map.options.scrollWheelZoom) {
         map.scrollWheelZoom.enable();
       }
     });
 
+    // using 'this._control' feels like a hack
+    this._geosearchClass.on('no-active-requests', function (e) {
+      L.DomUtil.removeClass(this._control._input, 'geocoder-control-loading');
```

Comment:
yeah, accessing private variables is usually a bad idea as they can disappear without notice.